react/week3/todo-list/src/Helper.js

```diff
@@ -0,0 +1,8 @@
+const getNextId = (todos) => {
+  const lastTodo = todos[todos.length - 1];
+  const nextId = lastTodo === undefined ? 1 : lastTodo.id + 1;
```

Comment:
If you want, you can make this slightly shorter:
```suggestion
  const nextId = lastTodo ? 1 : lastTodo.id + 1;
```